docs/src/main/sphinx/release/release-454.md

```diff
@@ -0,0 +1,88 @@
+# Release 454 (7 Aug 2024)
+
+## General
+
+* Add support for retrieving lightweight query information from the `QueryInfo`
+  endpoint with the new `pruned` property. ({issue}`20843`)
+* Improve performance of queries with multiple distinct aggregations. ({issue}`22355`)
+* Skip query completed notifications in the event listener when there are too
+  many concurrent completion requests. The maximum number of requests to handle
+  before skipping can be configured with the
+  `event-listener.max-concurrent-query-completed-events` configuration property. ({issue}`22911`)
+* Fix failure for queries involving {func}`json_parse` and a cast to `array`,
+  `map`, or `row`. ({issue}`22840`)
+
+## Security
+
+* Fix the Trino user incorrectly defaulting to the `user.name` property when no
+  user is provided. ({issue}`22903`)
+
+## Web UI
+
+* Add information about which tables and routines have been referenced by a
+  query. ({issue}`20843`)
+
+## JDBC driver
+
+* Add support for the `NO_GENERATED_KEYS` option in the
+  `Connection.prepareStatement` method. ({issue}`22100`)
+
+## BigQuery connector
+
+* Add support for the `execute` procedure. ({issue}`22692`)
+
+## Delta Lake connector
+
+* Add S3 security mapping for the native S3 filesystem. ({issue}`22559`)
+* Add support for caching table metadata to the metastore. ({issue}`21463`)
```

Comment:
Not sure what this means. What's the user-visible impact? cc @ebyhr

Reply:
This improves metadata listing speed from user perspective.